input/blog/2021-11-30-cake-v2.0.0-released.md

```diff
@@ -0,0 +1,226 @@
+
+See [Sunsetting of .NET Framework and .NET Core runners in Cake 2.0] for details.
+
+### Improvements to aliases for dotnet
+
+Cake 1.x had many [`DotNetCore***`](https://cakebuild.net/dsl/dotnetcore/) aliases for executing `dotnet` such as `DotNetCoreBuild`, `DotNetCoreTest`, etc.
+The `Core` part in the name existed for historical reasons when .NET Core was first introduced and lasted until v3.1.
+[With .NET 5, Microsoft dropped the "Core" from the name](https://devblogs.microsoft.com/dotnet/introducing-net-5/), and Cake 2.0 follows this naming schema for its .NET aliases:
+`DotNetCoreBuild` becomes `DotNetBuild`, `DotNetCoreTest` becomes `DotNetTest`, etc.
+
+:::{.alert .alert-info}
+Note that the old aliases are still available but deprecated and will be removed in Cake 3.0.
+:::
+
+See [#3341](https://github.com/cake-build/cake/issues/3341) for details.
+
+### Improved GitHub Actions integration
+
+Cake 2.0 comes with a few improvements to the GitHub Actions build provider:
+
+- Properties of files and paths are now of type `FilePath` and `DirectoryPath`
+- New `Environment` properties which returns runtime info, runner name, and workflow action path
+- `UploadArtifact` command to upload a local directory or file into a file container folder and create an artifact.
+- `SetEnvironmentVariable ` command to create or update an environment variable for any steps running next in a job.
+- `AddPath` command to add a directory to the system PATH variable and automatically make it available to all subsequent actions in the current job.
+
+### Improved Azure Pipelines integration
+
+Cake 2.0 comes with a few improvements to the Azure Pipelines build provider.
+See [Azure Pipelines build provider updates](/docs/getting-started/upgrade#azure-pipelines-build-provider-updates) for details.
+
+### Addin recommended Cake version set to 2.0
+
+We've now set the recommended version of [Cake.Core](https://www.nuget.org/packages/Cake.Core) for addins to target to 2.0.
+
+This won't break the build but you might see warnings like below when addins or modules are loaded.
+
+```bash
+The assembly 'Cake.FileHelpers, Version=4.0.1.0, Culture=neutral, PublicKeyToken=null'
+is referencing an older version of Cake.Core (1.0.0).
+```
+
+### Updated docker images
+
+Images are now continuously built for the last 10 stable versions of Cake version 1.0 or newer. Currently based on official Microsoft images available ( [mcr.microsoft.com/dotnet/sdk](https://mcr.microsoft.com/dotnet/sdk) ).
+
+To pin to a specific version suffix with Cake version i.e. cakebuild/cake:sdk-6.0 becomes cakebuild/cake:sdk-6.0-v2.0.0.
+
+Tags are added dynamically as new are added to Microsoft container registry, check [hub.docker.com/r/cakebuild/cake/tags](https://hub.docker.com/r/cakebuild/cake/tags) for currently available tags.
+
+Pull-requests and issues for the docker images can be raised in Cake's docker repo: [github.com/cake-build/docker](https://github.com/cake-build/docker)
+
```

Comment:
Should we add another chapter that Azure DevOps Build Task Extension has been deprecated? Or do we want to announce this in a separate blog post?

Reply:
Think we can do a separate follow-up one, if you have something ready you can add it as a suggestion.

Reply:
Perhaps we should we show all contributors here for Cake 1.0 to 2.0?

Reply:
> Perhaps we should show all contributors here for Cake 1.0 to 2.0?

@patriksvensson good idea, added.